clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -279,6 +294,13 @@ public void disconnected(ClientDescriptor clientDescriptor) {
 
   @Override
   public EhcacheEntityResponse invoke(ClientDescriptor clientDescriptor, EhcacheEntityMessage message) {
+    if (this.offHeapResourceIdentifiers.isEmpty()) {
+      return responseFactory.failure(
+          new IllegalStateException("Server started without any offheap resources defined." +
```

Comment:
You're right, I didn't notice you were outside the try - catch block. But you could simply move that code within it.

Maybe we should keep `ResourceConfigurationException` for client-server configuration mismatch, but in that case another exception that extends from `ClusteredEhcacheException` should be used.